pubspec.yaml

```diff
@@ -13,7 +13,7 @@ repository: https://github.com/privacyidea/pi-authenticator
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 3.0.8+0300008 # TODO Set the right version number
+version: 3.0.9+0300009 # TODO Set the right version number
```

Comment:
the current beta in the playstore has version 3.0.7 if im not mistaken. Therefore, i think we should set this to 3.0.8.